app/models/user.rb

```diff
@@ -95,6 +95,12 @@ class User < ApplicationRecord
           end
         }
 
+  def require_2sv?
+    return require_2sv unless organisation
+
+    organisation.require_2sv? || require_2sv
```

Comment:
I think we should still include whether someone is exempted here as part of the logic (so if (organisation requires AND not exempted) OR user requires 2sv)). 

I think there are potentially some bits of logic that would be unintentionally changed if we didn't include this condition, that won't be caught by tests at the moment since we don't widely use mandated 2sv orgs in test cases - for example, the behaviour of this method https://github.com/alphagov/signon/blob/main/app/models/user.rb#L323-L325

I think even if it didn't affect any existing behaviour, it'd be good protection against future use of this method, since I think I'd find it surprising it didn't take into account exemptions.

Reply:
Sure, that seems a reasonable thing to add. I'll look at this after lunch.